pwa/src/components/TreatmentForm.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { ReactElement } from 'react'
+import { TextareaItem, Accordion, Checkbox, List, InputItem  } from 'antd-mobile'
+import '../scss/Login.scss'
+import * as data from './TreatmentFields.json'
+
+/*
+
+Treatment Fields are stored in TreatmentFields.json in the format:
+
+{
+    "treatments" : [
+        {
+            "id": "multivitamins",
+            "group-title": "Multivitamins",
+            "group": [
+                { "name": "Multivitamins - Childrens", "id": "vitamin-childrens" },
+                { "name": "Multivitamins - Adult", "id": "vitamin-adult" },
+                { "name": "Multivitamins - Prenatal", "id": "vitamin-prenatal" }
+            ]
+        },
+        {...}
+    ]
+}
+*/
+
+const CheckboxItem = Checkbox.CheckboxItem;
+const treatmentFields = data.treatments
+
+export default function TreatmentForm(): ReactElement {
+    return (
+        <div id="treatment-form-wrapper" data-testid="treatment-form-wrapper">
+            {treatmentFields.map((field) => {
+                const formElements: ReactElement[] = []
+                    
+                if (field['text-input-title'] != null && field['text-input-id'] != null) {
+                    formElements.push(
+                        <List renderHeader={() => field['text-input-title']}>
```

Comment:
Same issue @meghanmc had with the click-able region -- it'd be easier if you could click anywhere on the span that contains the text in order to check the box rather than just the circle on the left of the screen.

Reply:
Fixed, new version was pushed.